### ISSUE TITLE
Projects with dev-containers should be grouped on the dashboard

### ISSUE THREAD

**Issue by DarylStark** [Jannchie/codetime-web-v3#3]
Projects on the dashboard should be grouped together if they are in a dev container, instead of being displayed as two different projects.

For instance, the following two are the same projects, but since I used a Dev Container one time and a Codespace the other time, the dashboard shows it as two different projects.

```
Project
my_rest_api [Dev Container: DevContainer - My REST API]  ... 16 hrs 8 mins
my_rest_api [Codespaces: ominous xylophone]              ... 4 hrs 46 mins
```

**Comment by Jannchie** [comment]
Indeed, project should now be renamed workspace... I should reconsider how to detect projects.
